app/pages/forms/[formId].vue

```
@@ -0,0 +1,146 @@
+
+const onComplete = async (args: ProjectApplicationForm) => {
+  if (!form.value) {
+    return
+  }
+  const formData = new FormData()
+
+  // Append non-file fields
+  for (const [key, value] of Object.entries(args)) {
+    if (Array.isArray(value) && value[0] instanceof File) {
+      // Append each file in attachment arrays
+      value.forEach((file: File) => formData.append(key, file))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)